data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,947 @@
+	autosave
+	landing
+	source "Hai-home"
+	clearance
+	to offer
+		has "Hai Leaks Response 4: done"
+	on offer
+		event "secret mountaintop"
+		event "role-play fleet deception"
+		# Hai like you a lot now, and as a result, you have permanent access to Hai-home:
+		"reputation: Hai" >?= 100
+		log `The working solution to the secrecy crisis for now is to have the Hai post a number of ships on the other side of the wormhole to play-act as an isolated furry role-play community. This is only a short-term solution. While they are working on a longer-term solution Sayari has quietly asked you to look into a rumour about more kidnapped Hai. She does not consider it to be a particularly likely scenario, but her contacts insist and so she would like you to be sure there's no issues lurking while the diplomatic process is working.`
+		conversation
+			`When you return to Hai-home you are met by a group of relieved diplomats.`
+			`	Of all of them Alondo seems to be the least tired and takes the time to let you know what's been going on, "Captain <last>! You've missed quite the palava. Still, I doubt you lost much by missing out on trying to explain to Sayari what a 'furry' is."`
+			choice
+				`	"Oh I don't know, could've been entertaining."`
+				`	"Hmm, yes, no probably not."`
+			`	He chuckles, "Anyway, I have a hunch that we're going to be needing you before too long, but for now they're putting some Hai patrols out on the other side of the wormhole with a couple of Navy observers on board to 'play the part', as it were, of an exclusive role-play community, just chilling out here and having fun."`
+			`	"Come though," he says, "I hear you've been instrumental in driving off some opportunistic pirates and helping stem the information leak, I'll buy you a drink on the other's behalf."`
+			`	Some hours later you leave the fancy cocktail bar Alondo took you to and begin making your way back to your ship to sleep, your cabin by far seems to be the best place to not be bothered.`
+			`	Part way there though a hooded figure suddenly falls in step beside you.`
+			choice
+				`	(Keep walking.)`
+				`	(Ask them who they are.)`
+			`	"Sssshhhh!" says the voice at your elbow, "do not react, and do not be alarmed, it is merely I."`
+			`	You look at the figure out of the corner of your eye and connect the whispered voice to the squirrelly muzzle poking out. To your surprise it is Sayari.`
+			`	"I should not be coming to you with this, so you must promise to say nothing of this to anyone just yet."`
+			choice
+				`	"Of course."`
+			`	"A very sweet friend of mine, who is much too enthusiastic and perhaps led by the vagaries of youth but nonetheless quite astute, keeps harping on to me about this theory of hers that a smuggling ring is kidnapping Hai and transporting them into human space."`
```

Comment:
```suggestion
			`	"A friend of mine is concerned about this theory of hers that a smuggling ring is kidnapping Hai and transporting them into human space."`
```
This is one spot where I think your specific writing style is too far removed from the rest of the game's narration. No offense intended, as I do like the way you write, but I do think things need to be consistent with the rest of the game's narration. I'd recommend rewriting much of Sayari's dialogue in the rest of this conversation accordingly. As such, the only edits I make for the rest of this conversation are grammar/spelling/punctuation, not style. There'd be too much I'd recommend changing that I'd be basically rewriting all of it.

Reply:
Sayari doesn't really get a lot of dialogue that's not strictly professional in the rest of the game. When she's not acting strictly in her professional role I was aiming for a 'divorced, middle-aged mum' vibe in her speech for personal interactions.
I am open to opinions on alternative styles though.

Reply:
Yeah, I think I see what you're going for. My main concern was with words like `vagaries`, `nonetheless`, and `harping`, because words of that type are pretty uncommon in American English. Maybe you could strike a balance between that vibe and keeping those types of words to a minimum? If you want me to suggest a rewrite since I'm the one who has concern with it I can, but if you'd rather tackle it yourself that's cool too. Your choice; let me know.

Reply:
Ah I see. If you'd like to attempt a re-write along those lines be my guest. Otherwise I'll see about it at some point.

Reply:
School's ramped up a good bit, so for the mean time feel free to do it yourself. Whenever things start to die down I'll be happy to suggest some things if you haven't done a rewrite already.

Reply:
This was a long time ago, but I don't particularly mind that style for Sayari's speech, I actually quite like that sentence, though it is a little long, as long as her speech is consistent with this as appropriate.